ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthGetLogs.java

```diff
@@ -103,7 +103,11 @@ public JsonRpcResponse response(final JsonRpcRequestContext requestContext) {
                 });
 
     if (ex.get() != null) {
-      LOG.debug("eth_getLogs call failed: ", ex.get());
+      LOG.atDebug()
+          .setMessage("eth_getLogs request {} failed: {}")
+          .addArgument(requestContext.getRequest())
```

Comment:
should there be a second addArgument?

Reply:
Good spot, I was printing the exception explicitly but don't need second arg after using the fluent api.